app/client/platforms/bedrock.ts

```
@@ -0,0 +1,703 @@
+    const requestTimeoutId = setTimeout(
+      () => controller.abort(),
+      REQUEST_TIMEOUT_MS,
+    );
+
+    try {
+      const res = await fetch(chatPath, {
+        method: "POST",
+        headers,
+        body: JSON.stringify({
+          ...requestPayload,
+          tools: tools && tools.length ? tools : undefined,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)